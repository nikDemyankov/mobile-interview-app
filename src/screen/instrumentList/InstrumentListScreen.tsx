import React, { memo } from 'react';
import { Text, View } from 'react-native';
import { Screen } from '../../component';

const InstrumentListScreen: React.FC = () => (
  <Screen>
    <View>
      <Text>Welcome</Text>
    </View>
  </Screen>
);

export default memo(InstrumentListScreen);
