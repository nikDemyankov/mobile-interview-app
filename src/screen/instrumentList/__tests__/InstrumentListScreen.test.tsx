import React from 'react';
import { create } from 'react-test-renderer';

import InstrumentListScreen from '../InstrumentListScreen';

jest.mock('../../../component', () => ({
  Screen: 'Screen',
}));

it('should render properly', () => {
  const component = create(<InstrumentListScreen />);
  expect(component).toMatchSnapshot();
});
