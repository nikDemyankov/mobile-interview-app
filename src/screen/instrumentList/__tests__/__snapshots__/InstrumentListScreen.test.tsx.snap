// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <View>
    <Text>
      Welcome
    </Text>
  </View>
</RCTSafeAreaView>
`;
